tests/test_client_request.py

```diff
@@ -981,7 +981,10 @@ async def test_content_encoding(  # type: ignore[misc]
         "post", URL("http://python.org/"), data="foo", compress="deflate", loop=loop
     )
     with mock.patch("aiohttp.client_reqrep.StreamWriter") as m_writer:
```

Comment:
In these cases, it's the mock.patch() that's missing it:
```suggestion
    with mock.patch("aiohttp.client_reqrep.StreamWriter", autospec=True, spec_set=True) as m_writer:
```

The other lines can then likely be removed entirely.